src/scripts.js

```diff
@@ -1,45 +1,44 @@
+// ************ IMPORTED FILES ***************
 import $ from 'jquery';
-// import users from './data/users-data';
-// import recipeData from  './data/recipe-data';
-// import ingredientData from './data/ingredient-data';
-
 import './css/base.scss';
 import './css/styles.scss';
 import './images/seasoning.png';
 import './images/apple-logo.png';
 import './images/apple-logo-outline.png';
 import './images/search.png';
 import './images/cookbook.png';
-
 import User from './user';
 import Recipe from './recipe';
 
+// ************ QUERY SELECTORS ***************
 let allRecipesBtn = document.querySelector(".show-all-btn");
 let filterBtn = document.querySelector(".filter-btn");
 let fullRecipeInfo = document.querySelector(".recipe-instructions");
 let main = document.querySelector("main");
-let menuOpen = false;
 let pantryBtn = document.querySelector(".my-pantry-btn");
-let pantryInfo = [];
-let recipes = [];
 let savedRecipesBtn = document.querySelector(".saved-recipes-btn");
 let searchBtn = document.querySelector(".search-btn");
 let searchForm = document.querySelector("#search");
 let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
 let tagList = document.querySelector(".tag-list");
-let user;
-let users;
+
+// ************ GLOBAL VARIABLES ***************
```

Comment:
Reorganizing the data like this in scripts is much easier to read now. Thank you!

Reply:
Yeah I like this new section, great addition for organization.